assets/js/blocks/product-search/index.tsx

```diff
@@ -95,18 +109,102 @@ registerBlockVariation( 'core/search', {
 			/>
 		),
 	},
-	// @ts-ignore waiting for @types/wordpress__blocks update
-	isActive: ( blockAttributes, variationAttributes ) => {
-		return (
-			blockAttributes.query?.post_type ===
-			variationAttributes.query.postType
-		);
-	},
 	category: 'woocommerce',
 	keywords: [ __( 'WooCommerce', 'woo-gutenberg-products-block' ) ],
 	description: __(
 		'A search box to allow customers to search for products by keyword.',
 		'woo-gutenberg-products-block'
 	),
-	attributes: PRODUCT_SEARCH_ATTRIBUTES,
+	supports: {
+		align: [ 'wide', 'full' ],
+		inserter: isBlockVariationAvailable ? false : true,
+	},
+	example: {
+		attributes: {
+			hasLabel: true,
+		},
+	},
+	attributes,
+	transforms: isBlockVariationAvailable
+		? {
+				to: [
+					{
+						type: 'block',
+						blocks: [ 'core/search' ],
+						transform: () => {
+							return createBlock(
+								'core/search',
+								PRODUCT_SEARCH_ATTRIBUTES
```

Comment:
Instead of using default values, could we try to use the same label and placeholder that were set in the old block? Ie, if the user set the old block as so:

![imatge](https://user-images.githubusercontent.com/3616980/181546535-5ca37bb6-bda3-42f4-9e28-7dd1b40bd8b5.png)

The title and placeholder are lost when converting the block into the variation.